packages/Chem/src/chem-searches.ts

```diff
@@ -85,9 +85,10 @@ function colInvalidated(col: DG.Column): Boolean {
     col.getTag(FING_COL_TAGS.invalidatedForVersion) == String(col.version));
 }
 
-async function _invalidate(molCol: DG.Column) {
+async function _invalidate(molCol: DG.Column, createMols: boolean) {
   if (!colInvalidated(molCol)) {
-    await (await getRdKitService()).initMoleculesStructures(molCol.toList());
+    if (createMols)
+      await (await getRdKitService()).initMoleculesStructures(molCol.toList());
     LastColumnInvalidated = molCol.name;
```

Comment:
LastColumnInvalidated - should be in camelCase

Reply:
Done